docs/_tutorials/pipeline.md

```diff
@@ -269,17 +269,17 @@ by DeepSpeed:
 * `partition_method="uniform"` balances the number of layers per stage.
 
 ### Memory-Efficient Model Construction
-Building a `Sequential` and providing it `PipelineModule` is a convenient way
+Building a `Sequential` container and providing it to a `PipelineModule` is a convenient way
 of specifying a pipeline parallel model. However, this approach encounters
-scalability issues for massive models. Starting from a `Sequential` allocates
-the model in CPU memory redundantly by every worker. A machine with 16 GPUs
+scalability issues for massive models. In this approach each worker replicates
+the whole model in CPU memory. For example, a machine with 16 GPUs
```

Comment:
Please verify that I have rewritten that without breaking the meaning. The current sentence "Starting from a `Sequential` allocates..." doesn't make much sense, so I dropped it altogether, but perhaps you can fix the original, so it'd be easier to understand why this approach does so.

Reply:
oh, I noticed I ended up with "this approach" in 2 subsequent sentences. Probably should refine that. But will wait for your feedback first.

Reply:
Good catch! What about combining the latter sentences into something like "However, this approach encounters scalability issues for massive models because each worker replicates the whole model in CPU memory." ?

Reply:
Perfect. Done.